report/render_report.R

```diff
@@ -1,67 +1,41 @@
 library(tidyverse)
 library(dotenv)
-library(REDCapR)
-library(lubridate)
-library(rmarkdown)
 library(sendmailR)
 library(redcapcustodian)
 library(argparse)
 
 init_etl("render_report")
 
 parser <- ArgumentParser()
-parser$add_argument("script_name", nargs=1, help="Script to be run")
+parser$add_argument("script_path", nargs=1, help="The full path of the script to be run")
+
```

Comment:
`render_report.R` works with a relative path. Maybe say this instead:

```suggestion
parser$add_argument("script_path", nargs=1, help="The path of the script to be run")

```

And do similarly in the copy of this file